packages/react-native-reanimated/plugin/src/jestMatchers.ts

```diff
@@ -68,27 +69,49 @@ expect.extend({
     };
   },
 
-  toIncludeInWorkletString(received: string, expected: string) {
-    // Regular expression pattern to find the `code` field in `initData`.
-    // @ts-ignore This regex works well in Jest.
-    const pattern = /code: "((?:[^"\\]|\\.)*)"/gs;
-    const matches = received.match(pattern);
+  toContainInWorkletString(received: string, expected: string) {
```

Comment:
Renamed `toInclude` -> `toContain` and added `toMatch` to align with Jest names.